### PR TITLE
Adds a Toy Which Makes Bwoink Sound

### PR DESCRIPTION
**Pull request by BlueHNT**
# Description

This adds a toy hammer based on the old BanHammer from ss13 which was a joke toy that made a bwoink sound. It also had a really old description which made minor fun of the russian translation changing it to 'bangammer' and 'reisin' which I decided to keep for the veterans.

---

<details><summary><h1>Media</h1></summary>
<p>

image
image

</p>
</details>

---

# Changelog

:cl:
- add: Added bwoink hammer with bwoink sound

### CODE REVIEW

File: Resources/Textures/Objects/Fun/bwoink_hammer.rsi/meta.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "taken from BigIron github",
```
Comment (stellar-novas) [inline Resources/Textures/Objects/Fun/bwoink_hammer.rsi/meta.json:4]: plz give repo link and commit hash
Link to the file at the commit also works

**Review by stellar-novas — CHANGES_REQUESTED** [review]

**Review by DangerRevolution — CHANGES_REQUESTED** [review]
Fix credit
Remove the comment yapping
Add this into the toy vendor as a mega-rare spawn

File: Resources/Textures/Objects/Fun/bwoink_hammer.rsi/meta.json (hunk 1 above)
Comment (BlueHNT) [reply Resources/Textures/Objects/Fun/bwoink_hammer.rsi/meta.json:4]: oh shit I forgot to edit this.

**Comment by BlueHNT** [comment]
Build and Test Debug just hates me

**Comment by DEATHB4DEFEAT** [comment]
> Build and Test Debug just hates me

Hates everyone

**Review by FoxxoTrystan — APPROVED** [review]

File: Resources/Audio/Admin/adminhelp_old.ogg
Comment (stellar-novas) [inline Resources/Audio/Admin/adminhelp_old.ogg:1]: Maybe rename to `bwoink.ogg` and move it? Since it isn't used for admin stuff now

**Review by stellar-novas — CHANGES_REQUESTED** [review]
Need to add attribution for `adminhelp_old.ogg` or whatever it's going to be called now.

File: Resources/Prototypes/Entities/Objects/Fun/prizeticket.yml [hunk 2]
```diff
@@ -392,4 +392,6 @@
       - id: ThronglerToy
         prob: 0.30
         orGroup: Prize
-        
\ No newline at end of file
+      - id: BwoinkHammer
+        prob: 0.30
+        orGroup: Prize
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Entities/Objects/Fun/prizeticket.yml:397]:
```suggestion
        orGroup: Prize

```

File: Resources/Prototypes/Entities/Objects/Fun/toys.yml [hunk 3]
```diff
@@ -1745,6 +1745,34 @@
         types:
           Blunt: 0
 
+- type: entity
+  parent: ToyHammer
+  id: BwoinkHammer
+  name: bwoink hammer
+  description: A toy hammer which makes a wildly terrifying sound. It is missing it's old spirit of bangammer and is still unreisinable.
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Entities/Objects/Fun/toys.yml:1752]:
```suggestion
  description: A toy hammer which makes a wildly terrifying sound. It is missing it's old spirit of bangammer and is still unraisinable.
```

File: Resources/Textures/Objects/Fun/bwoink_hammer.rsi/meta.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    {
+      "name": "inhand-left",
+      "directions": 4
+    },
+    {
+      "name": "inhand-right",
+      "directions": 4
+    }
+  ]
+}
```
Comment (DEATHB4DEFEAT) [inline Resources/Textures/Objects/Fun/bwoink_hammer.rsi/meta.json:22]:
```suggestion
}

```

**Review by DEATHB4DEFEAT — APPROVED** [review]

**Pull request merged**